taxonomies/ingredients.txt

```diff
@@ -15,6 +15,8 @@ synonyms:en:hydrolysed, hydrolyzed
 
 synonyms:en:pasteurised, pasteurized
 
+synonyms:fr:de production biologique, issu de l'agriculture biologique, 
```

Comment:
Do you have examples?

Remember that synonyms in the taxonomies are for words that are part of the ingredient (not the whole ingredient)